File: frontend-moviez-app/src/app/multiplex/model/multiplex.ts
export class Multiplex {
    multiplexId: String;
	multiplexName: String;
	city: String;
	numberOfScreens: number;
	setOfScreens: Set<any>;
	
    
    constructor(multiplexId?:String,multiplexName?:String,city?:String, numberOfScreens?:number, setOfScreens?: Set<any>){
		this.multiplexId = multiplexId;
		this.multiplexName = multiplexName;
		this.city = city;
		this.numberOfScreens = numberOfScreens;
		this.setOfScreens = setOfScreens;
	}
}

File: backend-moviez-app/search-service/src/main/java/com/application/search/model/Multiplex.java
package com.application.search.model;

import java.util.Set;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@EqualsAndHashCode
public class Multiplex {
	private String multiplexId;
	private String multiplexName;
	private String city;
	private int numberOfScreens;
	private Set<Integer> setOfScreens;
}
